client/web/reporter/src/components/EditorToolbar.vue

```diff
@@ -40,6 +40,7 @@
         size="lg"
         variant="light"
         :disabled="processing"
+        :processing="processingClone"
```

Comment:
Text shouldn't be visible when the button is processing, compare your implementation with the other ones, you'll spot the difference